modules/network-observability-enriched-flows.adoc

```diff
@@ -6,10 +6,11 @@
 [id="network-observability-enriched-flows_{context}"]
 = Export enriched network flow data
 
-You can send network flows to Kafka, IPFIX, or both at the same time. Any processor or storage that supports Kafka or IPFIX input, such as Splunk, Elasticsearch, or Fluentd, can consume the enriched network flow data.
+You can send network flows to Kafka, IPFIX, the Red Hat build of OpenTelemetry, or all three at the same time. For Kafka or IPFIX, any processor or storage that supports those inputs, such as Splunk, Elasticsearch, or Fluentd, can consume the enriched network flow data.
 
 .Prerequisites
-* Your Kafka or IPFIX collector endpoint(s) are available from Network Observability `flowlogs-pipeline` pods.
+* Your Kafka, IPFIX, or OpenTelemetry collector endpoint(s) are available from Network Observability `flowlogs-pipeline` pods.
+* If you are exporting to OpenTelemetry, you must install the Red Hat build of OpenTelemetry Operator.
```

Comment:
```suggestion
```

Reply:
@jpinsonneau should this prerequisite instead be, 
> If you are exporting to OpenTelemetry, you must have a compatible OpenTelemetry endpoint available, such as Red Hat build of OpenTelemetry, Jaeger, or Prometheus.

Or no prerequisite at all?

Reply:
```
Your Kafka, IPFIX, or OpenTelemetry collector endpoint(s) are available from Network Observability `flowlogs-pipeline` pods.
```
:arrow_up:  That one is still valid